namespace-change.adoc

```diff
@@ -0,0 +1,246 @@
+2.0 + Jakarta version of Deltaspike.
+
+- Micro Profile examples are broken due to the ongoing integration work
+of SmallRye impls. If the integration is done, we can fix them.
+
+- If Hibernate is used, we either need to upgrade to 6.0.0 _or_ use the
+Jakarta artifact from the 5.6.x series. In some examples, we still use
+Hibernate 4, so the upgrade also requires to deal with
+"tomee.jpa.factory.lazy".
+
+- Check for "http" repositories to avoid the default http blocker in
+newer Maven versions.
+
+== Artifacts
+
+|====
+|From | To | Comment
+
+a|
+[source, xml]
+----
+<plugin>
+  <groupId>org.apache.maven.plugins</groupId>
+  <artifactId>maven-compiler-plugin</artifactId>
+  <version>x.x.x</version>
+  <configuration>
+    <source>8</source>
+    <target>8</target>
+  </configuration>
+</plugin>
+----
+a|
+[source, xml]
+----
+<plugin>
+  <groupId>org.apache.maven.plugins</groupId>
+  <artifactId>maven-compiler-plugin</artifactId>
+  <version>3.10.1</version>
+  <configuration>
+    <source>11</source>
+    <target>11</target>
+  </configuration>
+</plugin>
+----
+b|
+Project master branch now targets Java 11.
+
+
+a|
+[source, xml]
+----
+<dependency>
+  <groupId>org.jboss.arquillian</groupId>
+  <artifactId>arquillian-bom</artifactId>
+  <version>x.x.x-.</version>
+  <scope>import</scope>
+  <type>pom</type>
+</dependency>
+----
+a|
+[source, xml]
+----
+<dependency>
+  <groupId>org.jboss.arquillian</groupId>
+  <artifactId>arquillian-bom</artifactId>
+  <version>1.7.0.Alpha10</version>
+  <scope>import</scope>
+  <type>pom</type>
+</dependency>
+----
+b|
+-
+
+
+a|
+[source, xml]
+----
+<groupId>javax.servlet</groupId>
+<artifactId>jstl</artifactId>
+<version>1.1.2</version>
+----
+a|
+[source, xml]
+----
+<groupId>jakarta.servlet.jsp.jstl</groupId>
+<artifactId>jakarta.servlet.jsp.jstl-api</artifactId>
+<version>2.0.0</version>
+----
+b|
+-
+
+
+a|
+[source, xml]
+----
+groupId>org.apache.myfaces.core</groupId>
+<artifactId>myfaces-api</artifactId>
+<version>2.1.8</version>
+----
+a|
+[source, xml]
+----
+groupId>org.apache.myfaces.core</groupId>
+<artifactId>myfaces-api</artifactId>
+<version>3.0.1</version>
+----
+b|
+-
+
+
+
+a|
+[source, xml]
+----
+<dependency>
+  <groupId>javax.websocket</groupId>
+  <artifactId>javax.websocket-api</artifactId>
+  <version>1.x</version>
+</dependency>
+----
+a|
+[source, xml]
+----
+<dependency>
+  <groupId>jakarta.websocket</groupId>
+  <artifactId>jakarta.websocket-api</artifactId>
+  <version>2.0.0</version>
+</dependency>
+----
+b|
+-
+
+
+a|
+[source, xml]
+----
+<dependency>
+    <groupId>org.apache.tomcat</groupId>
+    <artifactId>tomcat-api</artifactId>
+    <version>9.0.x</version>
+    <scope>provided</scope>
+</dependency>
+----
+a|
+[source, xml]
+----
+<dependency>
+    <groupId>org.apache.tomcat</groupId>
+    <artifactId>tomcat-api</artifactId>
+    <version>10.0.20</version>
+    <scope>provided</scope>
+</dependency>
+----
+b|
+Or latest Tomcat 10.0.x.
```

Comment:
"or latest Tomcat 10.0.x (depending on the Tomcat version used in the root pom.xml)" ?